src/GenericPhotosensor.cpp

```diff
@@ -54,12 +55,15 @@ void GenericPhotosensor::Construct()
 
   G4double window_thickness = thickness_/4.;
 
+  G4Material* OpSil = Materials::OpticalSilicone();
```

Comment:
For local variables, we're using this type of naming convention: `MyClass* my_class;`
Therefore, change `OpSil` for `op_sil` or `op_silicone`.